### ISSUE TITLE
Tree: p-tree doesn't pass Accessibility Insights for Web Fastpass

### ISSUE THREAD

**Issue by previta21** [primefaces/primeng#11740]
### Describe the bug

We are using p-tree and is failing Fastpass aria-required-parent rule. When we inspect from UI and start assigning role=’none’ to all the li elements of p-treenode inside p-tree from UI and again run the accessibility insights the error reduces. The expected output was achieved with the earlier version of Primeng. The current version does not have role assigned to the Li elements whereas the earlier version had role.

### Environment

Production

### Reproducer

https://www.primefaces.org/primeng/tree

### Angular version

13.3.1

### PrimeNG version

13.3.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v14.18.3

### Browser(s)

All

### Steps to reproduce the behavior

We have verified the issue on below Primeng URL
URL:(https://www.primefaces.org/primeng/tree)access

Steps to Reproduce:
1. Open the above URL-->source-->edit in stackblitz--> open in new window and run the accessibility insights.
2. Inspect html elements from browser.
3. Add role="none" to the Li elements of p-treenode inside p-tree and again run the accessibility insights. There will be no errors present.

### Expected behavior

All the li elements of p-treenode inside p-tree must have roles assigned to pass the aria-required-parent Web accessibility insights.

**Comment by cetincakiroglu** [comment]
Hi,

Accessibility is a major concern of the Prime UI libraries and PrimeNG is no exception. [PrimeTek](https://www.primetek.com.tr/) teams have initiated a significant process to review and enhance the accessibility features of the components. This guide documents the foundation of the general guidelines that PrimeNG will follow and each component documentation will have a separate Accessibility section that states the keyboard support, screen reader compatibility, the implementation details along with tips to achieve WCAG compliancy. This work has been initiated in Q2 2022 and planned to be completed by early Q3. PrimeNG will be the reference implementation which then will be ported to PrimeFaces, PrimeNG and PrimeVue.
